### ISSUE TITLE
Overload bypass

### ISSUE THREAD

**Issue by Math321** [LambdaInnovation/AcademyCraft#215]
I'm a Level 5 Teleporter. I haven't tried this with the other two ability types, but I'd assume this bug is universal.

If I spam some of my abilities fast enough back-to-back (Penetrate Teleport and Mark Teleport, for example) I can go past my maximum overload and still be able to use my abilities. Pretty much all my abilities seem to function correctly after going past maximum overload except Flashing. As soon as I use Flashing, the game realizes I'm above my maximum overload and prevents me from using my abilities until the Overload goes down to 0 again.

In short, going past your maximum Overload allows you to effectively ignore Overload altogether (though you do have to be careful to not go back under your maximum, or the glitch ends).

**Comment by WeAthFoLD** [comment]
I see, this is caused by network latency between C/S, the control judgement is done in client side but overload happens in server. There should possibly be some callback that terminates all the currently active skills when player is overloaded.

**Comment by Renniuq** [comment]
I've found that continuous abilities (like electromaster's magnet movement) seem to do a similar thing where they don't cut out when you reach max overload, and instead continue fine until they run out of cp or are stopped. Then they actually get stopped.

**Comment by WeAthFoLD** [comment]
I believe this is already fixed in 1.0pr3. I have rewritten the client control part so that when you overload, all skills are aborted.
